src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -246,7 +251,7 @@ private void InitializeOpenBuffers(SnapshotSpan triggerSpan)
             _triggerView.SetSelection(new SnapshotSpan(triggerSpan.Snapshot, startingSpan));
         }
 
-        this.UndoManager.CreateInitialState(this.ReplacementText, _triggerView.Selection, new SnapshotSpan(triggerSpan.Snapshot, startingSpan));
+        UndoManager.CreateInitialState(ReplacementText, _triggerView.Selection, new SnapshotSpan(triggerSpan.Snapshot, startingSpan));
```

Comment:
could you undo these?  `this.` allows us to see that these are accessing members on the instance.

importantly, without this, i don't know if UndoManager is a property, or a class that we're calling a static member on.

Reply:
done